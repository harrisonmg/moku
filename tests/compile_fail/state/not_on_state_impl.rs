mod blinky {
    use moku::*;

    struct Top {}

    struct Bottom {}

    #[superstate(Top)]
    impl Default for Bottom {
        fn default() -> Self {
            Self {}
        }
    }
}

fn main() {}
