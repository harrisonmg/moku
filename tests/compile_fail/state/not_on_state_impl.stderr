error: custom attribute panicked
 --> tests/compile_fail/state/not_on_state_impl.rs:8:5
  |
8 |     #[superstate(Top)]
  |     ^^^^^^^^^^^^^^^^^^
  |
  = help: message: not yet implemented
